cheat/cheatsheets/git

```diff
@@ -17,9 +17,16 @@ git commit -m "Your commit message"
 # To edit previous commit message
 git commit --amend
 
+# Git commit in the past
+git commit --date="`date --date='2 day ago'`"
```

Comment:
My Git (2.4.3) also supports these timespecs directly, so you could simplify this to `--date='2 days ago'`.

Not sure why, though… the manpage doesn’t mention it.